test/tic_tac_toe_test.exs

```diff
@@ -2,10 +2,10 @@ defmodule TicTacToeTest do
   use ExUnit.Case
   import ExUnit.CaptureIO
 
-
   describe "start/1" do
     test "(integration test) displays welcome message and board" do
```

Comment:
Is there a plan to move this test to maybe a `integration_test` directory perhaps?

Reply:
I'm actually not sure if there is a plan to do this or not, I think it makes sense and is a good point, especially if more integration tests are going to be added, but I don't know if we have a plan to do this yet 😶

Reply:
Yea I think we should find a home for integration tests in general. I don't think we'll need additional configuration for that but I could be wrong. Since you have more context than I do I'll leave it up to you to decide if you want to do that in this PR or punt to a follow PR.